src/main/resources/db/changelog/0004_create_table_category_restaurant.xml

```diff
@@ -4,21 +4,26 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                    https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
 
-    <changeSet id="0003-1" author="norbert">
+    <changeSet id="0004-1" author="kamil">
         <createTable tableName="restaurant_category">
-            <column name="id" type="BIGINT" autoIncrement="true">
-                <constraints nullable="false" primaryKey="true"/>
-            </column>
-            <column name="name" type="VARCHAR(150)">
-                <constraints unique="true"/>
-            </column>
+
             <column name="restaurant_id" type="BIGINT">
                 <constraints nullable="false"
-                             foreignKeyName="fk_category_restaurant_id"
+                             foreignKeyName="fk_category_restaurant_restaurant_id"
                              referencedTableName="restaurant"
                              referencedColumnNames="id"
```

Comment:
jak tam to tutaj dodać klucz główny czyli 
  primaryKey="true"
  primaryKeyName="pk_restaurant_category"